multi-node/vagrant/config.rb.sample

```diff
@@ -1,10 +1,26 @@
-#$update_channel="alpha"
+#
+# The VMs created for multi-node coreos-kubernetes on Vagrant is configured through
+# configuration options (global ruby variables) specified below. To override the
+# defaults that can be found in the Vagrantfile, copy this file to "config.rb", then
+# for each option that you want to change, uncomment the line (i.e., remove the
+# leading `#`) leaving just the `$`-prefixed variable name, and replace everything
+# after the equals sign with your desired setting.
+#
 
-#$controller_count=1
-#$controller_vm_memory=512
+# Official CoreOS channel from which updates should be downloaded: "alpha", "beta", "stable"
```

Comment:
Might be good to also mention minimum version (773.1.0+ ) here too (first version containing the kubelet binary)